capa/features/extractors/ghidra/extractor.py

```diff
@@ -37,7 +37,17 @@ def extract_file_features(self):
     def get_functions(self) -> Iterator[FunctionHandle]:
         import capa.features.extractors.ghidra.helpers as ghidra_helpers
 
-        yield from ghidra_helpers.get_function_symbols()
+        imports = ghidra_helpers.get_file_imports()
+        externs = ghidra_helpers.get_file_externs()
+        fakes = ghidra_helpers.map_fake_import_addrs()
```

Comment:
Let's move these to `__init__` to ensure the overhead is incurred only once per `GhidraFeatureExtractor` creation.